src/App.php

```diff
@@ -578,9 +578,9 @@ public function run()
             $this->html->template->dangerouslyAppendHtml('HEAD', $this->html->getJs());
             $this->is_rendering = false;
 
-            if (isset($_GET[Callback::URL_QUERY_TARGET]) && $this->catch_runaway_callbacks) {
+            if (isset($_GET['__atk_callback']) && $this->catch_runaway_callbacks) {
```

Comment:
Is this wanted change?

Reply:
No, again resolve conflict screw up. I put it back. Thanks